temporal-server.yaml

```diff
@@ -51,22 +59,46 @@ subpackages:
           install -Dm755 temporal-cassandra-tool "${{targets.subpkgdir}}"/usr/bin/temporal-cassandra-tool
       - uses: strip
 
+  - name: temporal-cassandra-tool-compat
+    description: "Compat package for temporal-cassandra-tool"
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}"/usr/local/bin
+          ln -s /usr/bin/temporal-cassandra-tool "${{targets.subpkgdir}}"/usr/local/bin
+      - uses: strip
+
   - name: tdbg
     description: "tdbg"
     pipeline:
       - runs: |
-          mkdir -p "${{targets.subpkgdir}}"/usr/bin
+          mkdir -p "${{targets.subpkgdir}}"/usr/local/bin
           install -Dm755 tdbg "${{targets.subpkgdir}}"/usr/bin/tdbg
```

Comment:
you created a /usr/local/bin in the previous step but you install the package into /usr/bin, this doesn't sound correct, am I right?